samples/colab/tflite_text_classification.ipynb

```diff
@@ -44,7 +44,7 @@
       "outputs": [],
       "source": [
         "%%capture\n",
-        "!python -m pip install iree-compiler iree-runtime iree-tools-tflite -f https://github.com/iree-org/iree/releases/latest\n",
+        "!python -m pip install iree-compiler iree-runtime iree-tools-tflite -f https://github.com/iree-org/iree/releases\n",
```

Comment:
Testing on this branch failed: https://buildkite.com/iree/iree-samples/builds/439
but https://colab.research.google.com/github/ScottTodd/iree/blob/samples-fix-vm/samples/colab/tflite_text_classification.ipynb works, so that might resolve itself after merging